docs: add faqs (#474)
sveltejs/cli: documentation/docs/10-introduction/20-faq.md

---
title: Frequently asked questions
---

> ## How do i run the `sv` CLI?

Reviewer: ```suggestion
## How do I run the `sv` CLI?
```

> ## How do i run the the `sv` cli?

Reviewer 2: ```suggestion
## How do i run the `sv` CLI?
```

Running the `sv` cli differs for each package manager. Here is a list of the most common commands:

- **NPM** : `npx sv create`

> - **PNPM** : `pnpm dlx sv create`

Reviewer 2: ```suggestion
- **PNPM** : `pnpx sv create` or `pnpm dlx sv create`
```

- **Yarn** : `yarn dlx sv create`
- **Bun** : `bunx sv create`
- **Deno** : `deno run npm:sv create`

## `npx sv` is not working

Some package mangers prefer to run locally installed tools instead of downloading and executing packages from the registry. This issue mostly occurs with `npm` and `yarn`. This usually results in an error message or looks like the command you were trying to execute did not do anything.

> Here is a list of issues with possible solution, users have encountered in the past

Reviewer 2: ```suggestion
Here is a list of issues with possible solutions that users have encountered in the past:
```

> - [`npx sv` create does nothing](https://github.com/sveltejs/cli/issues/472)

Reviewer 2: Is the solution different for each of these issues? If they share a solution it might be nice to document it here, but maybe that gets a little more difficult if there are three to document

Author: At least two of this issues have different solutions and one of them presents multiple workarounds. That's why i decided only linking those issues instead of incorporating the different possible solutions

> - [Document `sv` command name collision with `runit`](https://github.com/sveltejs/cli/issues/259)

Reviewer 2: ```suggestion
- [`sv` command name collides with `runit`](https://github.com/sveltejs/cli/issues/259)
```
